### PR TITLE
Nivogt/people picker updates

### PR DESCRIPTION
**Pull request by vogtn**
Updating people-picker with "Selected People" section. This allows developers to access the people-picker component and dynamically add selected-people.

### CODE REVIEW

**Comment by vogtn** [comment]
@nmetulev 

Here's the draft for the "Selected-People" section of the people-picker. Let me know if you require any changes. 

Thanks!

File: concepts/toolkit/components/people-picker.md [hunk 1]
```diff
@@ -30,13 +30,51 @@ By default, the `mgt-people-picker` component fetches events from the `/me/peopl
...
+![mgt-people-picker](./images/selected-people.png)
+
+Selected People data is accessible to populate via the following methods:
+
+***If no user is found for the provided `id`, no data will be rendered for that specific `id`**
+
+1. Giving an array of `person` object to the `selected-people` property. 
+
+      * Example
+    ```html
```
Comment (nmetulev) [inline concepts/toolkit/components/people-picker.md]: not sure I like this example, it's not a realistic scenario and not something a dev would do.

I'd use `document.querySelector('mgt-people-picker').selectedPeople = personObject;` where personObject is a User or Person from the graph

Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: Updated: 
439e606

File: concepts/toolkit/components/people-picker.md [hunk 2]
```diff
@@ -30,13 +30,44 @@ By default, the `mgt-people-picker` component fetches events from the `/me/peopl
...
 
+## Selected People
+
+The selected people section of the component renders each person chosen by developer or user. 
+
+![mgt-people-picker](./images/selected-people.png)
+
+Selected People data is accessible to populate via the following methods:
+
+***If no user is found for the provided `id`, no data will be rendered for that specific `id`**
```
Comment (nmetulev) [inline concepts/toolkit/components/people-picker.md]: This should move down as it is only for 2

File: concepts/toolkit/components/people-picker.md [hunk 3]
```diff
@@ -30,13 +30,44 @@ By default, the `mgt-people-picker` component fetches events from the `/me/peopl
...
+
+1. Setting the `selectedPeople` property directly:  
+
+      * Example
+    ```javascript
+    // personObject = User or Person from Microsoft Graph
+    document.querySelector('mgt-people-picker').selectedPeople = personObject;
```
Comment (nmetulev) [inline concepts/toolkit/components/people-picker.md]: This should be `selectedPeople.push(personObject)`

**Comment by nmetulev** [comment]
Moved the target branch to 'microsoft-graph-toolkit-staging'. Let's use that branch for staging all docs for next release

File: concepts/toolkit/components/people-picker.md (hunk 3 above)
Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: updated
aa1fbe3

File: concepts/toolkit/components/people-picker.md (hunk 2 above)
Comment (vogtn) [reply concepts/toolkit/components/people-picker.md]: Moved down: 

aa1fbe3

**Pull request merged**